lib/onceover/controlrepo.rb

```diff
@@ -345,10 +345,13 @@ def fixtures
     end
 
     def hiera_config_file
-      # try to find the hiera.yaml file
-      hiera_config_file = File.expand_path('./hiera.yaml', @spec_dir) if File.exist?(File.expand_path('./hiera.yaml', @spec_dir))
-      hiera_config_file = File.expand_path('./hiera.yaml', @root)     if File.exist?(File.expand_path('./hiera.yaml', @root))
-      hiera_config_file
+      case
+      when File.exist?(File.expand_path('./hiera.yaml', @spec_dir))
+        return File.expand_path('./hiera.yaml', @spec_dir)
+      when File.exist?(File.expand_path('./hiera.yaml', @root))
+        return File.expand_path('./hiera.yaml', @root)
+      end
+      nil
```

Comment:
@jessereynolds I couldn't wrap my head around it because it's possible that both will be true and therefore it will work in the opposite way as intended, but adding the return statements should do what we want